### PR TITLE
[SMALLFIX] Add javadoc for AccumulatingReducer#reduce

### PR DESCRIPTION
**Pull request by yindaren**
Add javadoc for AccumulatingReducer#reduce

### CODE REVIEW

File: tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java [hunk 1]
```diff
@@ -59,6 +59,15 @@ public AccumulatingReducer() {
     LOG.info("Starting AccumulatingReducer on " + mHostname);
   }
 
+  /**
+   * This method accumulates values based on their type
```
Comment (gpang) [inline tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java]: "Accumulates values based on their type."

(needs a period at the end of the sentence)

File: tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java [hunk 2]
```diff
@@ -59,6 +59,15 @@ public AccumulatingReducer() {
...
+   *
+   * @param key the type of values.
```
Comment (gpang) [inline tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java]: param requires no period at the end of the line

File: tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java [hunk 3]
```diff
@@ -59,6 +59,15 @@ public AccumulatingReducer() {
...
+   * @param values the values to accumulates.
```
Comment (gpang) [inline tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java]: param requires no period at the end of the line

Comment (yindaren) [inline tests/src/test/java/alluxio/hadoop/fs/AccumulatingReducer.java]: Thank you for reminding

**Pull request merged**